### PR TITLE
Fixed errors in cloud.rst on keys for fsspec

### PR DESCRIPTION
**Pull request by youtsumi**
### Description

The documentation on fsspec has some errors on keynames. I also fixed an error in an example. 

This pull request is to address the errors in the documentation. 

Fixes #<Issue Number>

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: docs/io/fits/usage/cloud.rst [hunk 1]
```diff
@@ -240,14 +240,14 @@ will attempt to download data in large contiguous blocks using a buffered
...
+``default_block_size`` and ``default_cache_type`` parameters to `fsspec.open`.  You can pass
 these parameters via the ``fsspec_kwargs`` argument of `astropy.io.fits.open`.
 For example, we can configure fsspec to make buffered reads with a minimum
-``block_size`` of 1 MB as follows:
+``default_block_size`` of 1 MB as follows:
 
 .. doctest-requires:: fsspec
 
-    >>> fsspec_kwargs = {"block_size": 1_000_000, "cache_type": "bytes"}
+    >>> fsspec_kwargs = {"default_block_size": 1000000, "default_cache_type": "bytes"}
```
Comment (pllim) [inline docs/io/fits/usage/cloud.rst:250]: Hmm if this was a typo, why didn't this fail the doctest?

Comment (youtsumi) [reply docs/io/fits/usage/cloud.rst:250]: Actually, the part of number was not an error. It is allowed in https://peps.python.org/pep-0515/ . So we don't need to change 1_000_000 to 1000000 but the other keys need to be fixed.

**Comment by saimn** [comment]
Could you explain which errors you are getting ?
I ran the example and it seems to works as expected, and from what I can see https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.http.HTTPFileSystem expects `block_size` and `cache_type`.

**Comment by youtsumi** [comment]
With s3fs '2023.10.0', if I gave `block_size` I got this error. But if gave `default_block_size`, it works fine. 
```
TypeError: AioSession.__init__() got an unexpected keyword argument 'block_size'
```

**Comment by saimn** [comment]
But this example does not use the s3 url, it uses `url` which is an http URL. That probably explains the error ?

**Comment by youtsumi** [comment]
You are probably right. Then this PR is no longer relevant.

**Comment by pllim** [comment]
Closing without merge then. Thanks anyway!

**Pull request closed without merging**